src/settings/AudioSettings.tsx

```diff
@@ -200,10 +229,37 @@ export default function GeneralSettings(props: ISettingsPanelProps) {
           </IconButton>
         </Tooltip>
       </Box>
+      <Box component="span" sx={{ display: 'flex', alignItems: 'center' }}>
+        <FormControlLabel
+          control={
+            <Switch
+              sx={switchStyle}
+              checked={forceMono}
+              name={forceMonoConfigKey}
+              onChange={(isForceMono) => {
+                setForceMono(Boolean(isForceMono.target.checked));
+                onChange(isForceMono);
+              }}
+            />
+          }
+          label="Force Mono"
```

Comment:
I'd maybe go with "Mono Input" rather than "Force Mono" here

Reply:
Yeah, agreed "Force Mono" isn't very clear.

Reply:
This one still needing addressed?